### ISSUE TITLE
error using module.run -> saltutil.runner -> state.orchestrate: "The following arguments are missing: _fun"

### ISSUE THREAD

**Issue by rhansen** [saltstack/salt#31851]
### Description of Issue

If I create a state file that uses `module.run` to execute `saltutil.runner` to execute `state.orchestrate`, I get the following error:

```
master:
----------
          ID: orchestrate
    Function: module.run
        Name: saltutil.runner
      Result: False
     Comment: The following arguments are missing: _fun
     Started: 18:17:12.237399
    Duration: 0.817 ms
     Changes:   
```

### Setup

  * `/etc/salt/master`:

    ```yaml
    file_roots:
      base:
        - /salt/base/states
    pillar_roots:
      base:
        - /salt/base/pillar
    module_dirs:
      - salt/modules
    ```

  * `/salt/base/states/foo.sls`:

    ```yaml
    orchestrate:
      module.run:
        - name: saltutil.runner
        - m_name: state.orchestrate
        - kwargs:
            mods: orchestrate
    ```

  * `/salt/base/states/orchestrate.sls`:

    ```yaml
    uptime:
      salt.function:
        - tgt: '*'
        - name: cmd.run
        - arg:
          - uptime
    ```

### Steps to Reproduce Issue

Configure as described above and run the following on the master:

    sudo salt master state.sls foo

(this assumes the master is also a minion with minion ID `master`)

### Versions Report

```
Salt Version:
           Salt: 2015.8.7
 
Dependency Versions:
         Jinja2: 2.7.2
       M2Crypto: Not Installed
           Mako: 0.9.1
         PyYAML: 3.10
          PyZMQ: 14.4.0
         Python: 2.7.6 (default, Jun 22 2015, 17:58:13)
           RAET: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.0.4
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: 1.5
          gitdb: 0.5.4
      gitpython: 0.3.2 RC1
          ioflo: Not Installed
        libgit2: Not Installed
        libnacl: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.3.0
   mysql-python: 1.2.3
      pycparser: Not Installed
       pycrypto: 2.6.1
         pygit2: Not Installed
   python-gnupg: Not Installed
          smmap: 0.8.2
        timelib: Not Installed
 
System Versions:
           dist: Ubuntu 14.04 trusty
        machine: x86_64
        release: 4.2.0-30-generic
         system: Ubuntu 14.04 trusty
```

**Comment by rhansen** [comment]
I just noticed the following in the documentation for [saltutil.runner](https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.saltutil.html#salt.modules.saltutil.runner):

>     salt.modules.saltutil.runner(_fun, **kwargs)
>         ...
>         name
>             The name of the function to run

Notice how the parameter name in the function signature is `_fun` but the documentation uses `name`.

If I change my `foo.sls` to this then it works:

```yaml
orchestrate:
  module.run:
    - name: saltutil.runner
    - _fun: state.orchestrate
    - kwargs:
        mods: orchestrate
```

**Comment by jfindlay** [comment]
@rhansen, thanks for reporting.

**Comment by rallytime** [comment]
@rhansen I have fixed the doc discrepancy in #32445. Thanks for letting us know and for the useful issue description!
